pep-0701.rst

```diff
@@ -302,33 +304,85 @@ will be tokenized as::
     NAME - 'a'
     PLUS - '+'
     NAME - 'b'
+    OP - ':'
+    FSTRING_MIDDLE - '.3f'
     RBRACE - '}'
     FSTRING_MIDDLE - ' more words '
     LBRACE - '{'
     NAME - 'c'
     PLUS - '+'
     NAME - 'd'
+    OP - '='
     RBRACE - '}'
-    FSTRING_END - ' final words' (without the end quote)
+    FSTRING_MIDDLE - ' final words'
+    FSTRING_END - "'"
 
 while ``f"""some words"""`` will be tokenized simply as::
 
     FSTRING_START - 'f"""'
-    FSTRING_END - 'some words'
-
-One way existing lexers can be adapted to emit these tokens is to incorporate a stack of "lexer modes"
-or to use a stack of different lexers. This is because the lexer needs to switch from "regular Python
-lexing" to "f-string lexing" when it encounters an f-string start token and as f-strings can be nested,
-the context needs to be preserved until the f-string closes. Also, the "lexer mode" inside an f-string
-expression part needs to behave as a "super-set" of the regular Python lexer (as it needs to be able to 
-switch back to f-string lexing when it encounters the ``}`` terminator for the expression part as well
-as handling f-string formatting and debug expressions). Of course, as mentioned before, is not possible to
-provide a precise specification of how this should be done as it will depend on the specific implementation
-and nature of the lexer to be changed.
-
-The specifics of how (or if) the ``tokenize`` module will emit these tokens (or others) and what
-is included in the emitted tokens are left out of this document and must be decided later in a regular
-CPython issue.
+    FSTRING_MIDDLE - 'some words'
+    FSTRING_END - '"""'
+  
+Changes to the tokenize module
```

Comment:
Should we add a link from "Backwards Compatibility" section to here to mention the fact that this change is indeed creates incompatible behavior for users of the internal CPython APIs such as `tokenize` module (right now it specify there is no compatibility issue for users, which I guess would be changed now)?

Reply:
Yeah, that’s a good point. I will be AFK sone days. Could you push something to this PR?

CC: @lysnikolaou what do you think?

Reply:
Yup, that makes sense.

Reply:
```suggestion
.. _701-tokenize-changes:

Changes to the tokenize module
```

Here's a ref target label for you (usage: `` :ref:`701-tokenize-changes` `` or `` :ref:`custom link title <701-tokenize-changes>` ``)

Reply:
> Yeah, that’s a good point. I will be AFK sone days. Could you push something to this PR?

Sure! Will push the changes for that section tomorrow.